lib/python/operations.cpp

```diff
@@ -80,6 +81,83 @@ void bind_midpoints(py::module &m) {
   });
 }
 
+void bind_experimental(py::module &m) {
+  m.def("experimental_transform", [](py::object const &kernel,
+                                     const Variable &a) {
+    auto fptr_address = kernel.attr("address").cast<intptr_t>();
+    auto fptr = reinterpret_cast<double (*)(double)>(fptr_address);
+    return variable::transform<double>(
+        a,
+        overloaded{core::transform_flags::expect_no_variance_arg<0>,
+                   [&kernel](const units::Unit &x) {
+                     py::gil_scoped_acquire acquire;
+                     return py::cast<units::Unit>(kernel.attr("unit_func")(x));
+                   },
+                   [fptr](const auto &x) { return fptr(x); }},
+        "custom");
```

Comment:
We can pass the name as an argument fro Python. Could just use the name of the input function.

Reply:
Good idea.